### PR TITLE
Link to dcrdata for blockchain size

### PR DESCRIPTION
**Pull request by noahpierau**
This PR closes issue #546

### CODE REVIEW

**Comment by jholdstock** [comment]
I'm happy with this changes, but it would be great to understand the discrepancy between 3.3GB and 2.4GB blockchain size before we merge it.

The 3.3GB figure came from running decrediton on a local machine and checking the size of the blockchain db stored on disk. It would be interesting to understand how dcrdata calculates the chain size, especially the historic size.

**Comment by papacarp** [comment]
The size as reported on dcrdata is a sum of all the individual block sizes.    As an example, this block is 6.2kB (https://explorer.dcrdata.org/block/275632).  We just sum all entries in our database for block size and report the result.  Which is approximately 2.6G (2,674,813,446).  Our database has recently started collecting blocks that are not valid and not part of the main chain as well and as of now those are also included in our totals.  The sum of all invalid and non-mainchain blocks is only 39KB however, so not a big deal.

**Comment by chappjc** [comment]
To reiterate @papacarp, the size dcrdata reports is the sum of the sizes of all serialized blocks, as opposed to the disk space occupied by the database.  There is quite a bit of metadata and other DB overhead that adds to the size on disk.

**Comment by noahpierau** [comment]
Based on the feedback I changed the final sentences. I also added the notion of SPV, since it doesn't require a full blockchain download. This will make the information more future-proof.

File: docs/getting-started/user-guides/decrediton-setup.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Last updated for v1.2.1.
 
 !!! info "Blockchain Download"
 
-	Decrediton needs to download a full copy of the Decred blockchain before it can be used. This means that a large download must be completed and a large amount of storage space must be be available on the device where Decrediton is intalled. You can check the current size of the Decred blockchain via [dcrdata charts](https://explorer.dcrdata.org/charts#blockchain-size). Total required storage space for Decrediton may vary, depending on your device.
+	Without SPV enabled, Decrediton needs to download a full copy of the Decred blockchain before it can be used. This means that a large download must be completed and a large amount of storage space must be be available on the device where Decrediton is intalled. You can check the current size of all serialized blocks via [dcrdata charts](https://explorer.dcrdata.org/charts#blockchain-size). The total size of Decrediton will be bigger, since additional wallet data is stored on your device.
```
Comment (jholdstock) [inline docs/getting-started/user-guides/decrediton-setup.md]: Typo: intalled -> installed.

**Comment by noahpierau** [comment]
Fixed. Not sure how I missed that :smile:

**Pull request merged**